include/deal.II/lac/affine_constraints.h

```diff
@@ -56,19 +56,6 @@ namespace internals
 template <typename number>
 class AffineConstraints;
 
-/**
- * ConstraintMatrix has been renamed to AffineConstraints. Provide a
- * compatibility typedef that defaults to AffineConstraints<double>.
- *
- * @deprecated Use AffineConstraints<double> instead of ConstraintMatrix
- */
-using ConstraintMatrix = AffineConstraints<double>;
-// Note: Unfortunately, we cannot move this compatibility typedef into
-// constraint_matrix.h directly. This would break a lot of user projects
-// that include constraint_matrix.h transitively due to various deal.II
-// headers that include the file.
```

Comment:
@masterleinad How do you fix this particular problem then?
```

```
// Note: Unfortunately, we cannot move this compatibility typedef into
// constraint_matrix.h directly. This would break a lot of user projects
// that include constraint_matrix.h transitively due to various deal.II
// headers that include the file.